support/routes/v2/isobot/index.ts

```diff
@@ -45,3 +78,112 @@ bot.command("/siteup", async ({ payload, respond, ack }) => {
 
   return botService.dnsChecker(payload).map((response) => respond(response))
 })
+
+bot.command(
+  "/clone",
+  validateIsomerUser,
+  async ({ command, ack, respond, payload, client }) => {
+    await ack()
+
+    const HELP_TEXT =
+      "Usage: `/clone <github_repo_name>`. Take note that this locks the repo for 15 minutes by default. To bypass this behaviour, add `-n` at the end of the command"
+    const tokens = command.text.split(" ")
+
+    const hasUnrecognisedLastToken = tokens.length === 2 && tokens[1] !== "-n"
+    if (tokens.length < 1 || hasUnrecognisedLastToken) {
+      return respond(HELP_TEXT)
+    }
+
+    // NOTE: Invariant maintained:
+    // 1. we always need 0 < tokens.length < 3
+    // 2. token at index 0 is always the github repository
+    // 3. token at index 1 is always the "-n" option
+    const isHelp = tokens[0].toLowerCase() === "help"
+    if (isHelp) {
+      return respond(HELP_TEXT)
+    }
+
+    const repo = tokens[0]
+    const shouldLock = tokens.length === 2 && tokens[1] === "-n"
+    await client.chat.postMessage({
+      channel: BOT_AUDIT_CHANNEL_ID,
+      text: `${payload.user_id} attempting to clone repo: ${repo} to EFS. Should lock: ${shouldLock}`,
+    })
+
+    const base = shouldLock ? repairService.lockRepo(tokens[0]) : okAsync("")
+    return base
+      .andThen(repairService.cloneRepo)
+      .map(() => respond(`${repo} was successfully cloned to efs!`))
+      .mapErr((e) => respond(`${e} occurred while cloning repo to efs`))
+  }
+)
+
+bot.command(
+  "/lock",
+  validateIsomerUser,
+  async ({ command, ack, respond, payload, client }) => {
+    await ack()
+
+    const HELP_TEXT =
+      "Usage: `/lock <github_repo_name> -d <duration_in_minutes>`. Take note that this locks the repo for 15 minutes by default if `-d` is not specified"
```

Comment:
should d also have a max time? what if someone puts in an accidental 10000000 mins or something?

Reply:
makes sense, will make this change